locales/nb_NO/pcsx2_Main.po

```diff
@@ -1208,21 +1208,22 @@ msgid ""
 "The system default should be fine for most operating systems."
 msgstr ""
 "Skift språket bare hvis du trenger det.\n"
-"Systemet sin standardinnstilling bør gå greit for de fleste operativsystemer."
+"Systemet sin standardinnstilling burde være greit for de fleste "
+"operativsystemer."
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:95
 msgid "Welcome to PCSX2!"
 msgstr "Velkommen til PCSX2!"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
 #, fuzzy
+#| msgid "Configuration Guides (online)"
```

Comment:
In poedit, you need to remove the "needs work" status. Otherwise it'll be untranslated.